learn/quickstart/operators_control_flow.md

```diff
@@ -119,19 +119,104 @@ __Example:__ `do` loop with skip
 
 ```fortran
   integer :: i
-  do i=1,10,2    
+  do i=1,10,2
     print *, i   ! Print odd numbers
   end do
 ```
 
+### `do while()`
 
-__Example:__ `do while` loop
+A condition may be added to a `do` loop with the `while` keyword. The loop will be executed while the condition given
+in `while()` evaluates to `.true.`.
+
+__Example:__ `do while()` loop
 
 ```fortran
   integer :: i
   i = 1
-  do while (i<11)   
+  do while (i<11)
     print *, i
     i = i + 1
   end do
-```
\ No newline at end of file
+  ! Here i = 11
+```
+
+### `do concurrent()`
+
+The `do concurrent` loop is used to specify that the _inside of the loop has no interdependencies_. By this, it is indicated
+that any loop does not depend on the loops before it. It is also necessary that any state changes that may occur must only
+happen within each `do concurrent` loop. Because of these requirements, it is important to be carful with what is written inside the loop.
+
+However, the true nature of the `do concurrent` construct may not be fully understood by the paragraph above. `do concurrent`
+was specifically created to tell the compiler that it may parallelize/_SIMD_ what is inside the `do concurrent`,
+which could potentially lead to faster code. The intention of the programmer is also conveyed more clearly.
```

Comment:
```suggestion
```

I've moved this information to the first paragraph in my previous suggestion - I think the motivation for `do concurrent` is better stated upfront.